lib/tasks/pulmap.rake

```diff
@@ -8,7 +8,7 @@ task :ci do
   SolrWrapper.wrap(shared_solr_opts.merge(port: 8985, instance_dir: 'tmp/pulmap-core-test')) do |solr|
     solr.with_collection(name: "pulmap-core-test", dir: Rails.root.join("solr", "conf").to_s) do
       system 'RAILS_ENV=test rake geoblacklight:index:seed'
-      system 'RAILS_ENV=test rake spec'
```

Comment:
why was this failing?

Reply:
I don't know, but Circle does not register failing specs unless you invoke the spec task via ``Rake::Task['spec'].invoke`. I tested and it fails properly this way.